airflow/providers/google/cloud/hooks/datafusion.py

```diff
@@ -387,8 +398,9 @@ def get_pipeline_workflow(
             quote(pipeline_id),
         )
         response = self._cdap_request(url=url, method="GET")
-        if response.status != 200:
-            raise AirflowException(f"Retrieving a pipeline state failed with code {response.status}")
+        self._check_response_status_and_data(
+            response, f"Retrieving a pipeline state failed with " f"code {response.status}"
```

Comment:
it looks like you split string manually and then black moved it to one line. There is " f" now

Reply:
yes, i know
i am fixing now it :)